examples/cfr_export_demo.rs

```diff
@@ -0,0 +1,97 @@
+use rs_poker::arena::cfr::{export_cfr_state, CFRState, NodeData, PlayerData, TerminalData};
+use rs_poker::arena::GameState;
+use std::path::Path;
+
+/// This example demonstrates the export functionality of the CFR state.
+/// It creates a simple CFR tree and exports it to DOT, PNG, and SVG formats.
+fn main() {
+    // Create a game state with 2 players
+    let game_state = GameState::new_starting(vec![100.0; 2], 10.0, 5.0, 0.0, 0);
+    let mut cfr_state = CFRState::new(game_state);
+    
+    // Root -> Player 0 decision
+    let player0_node = NodeData::Player(PlayerData { regret_matcher: None });
+    let player0_idx = cfr_state.add(0, 0, player0_node);
+    
+    // Player 0 fold
+    let terminal_fold = NodeData::Terminal(TerminalData::new(-10.0));
+    let _fold_idx = cfr_state.add(player0_idx, 0, terminal_fold);
+    
+    // Player 0 call
+    let player0_call = cfr_state.add(player0_idx, 1, NodeData::Chance);
+    
+    // Player 0 raise
+    let player0_raise = cfr_state.add(player0_idx, 2, NodeData::Chance);
+    
+    // After call - chance node (dealing flop)
+    for i in 0..3 {
+        // Create 3 sample card possibilities
+        let player1_node = NodeData::Player(PlayerData { regret_matcher: None });
+        let player1_idx = cfr_state.add(player0_call, i, player1_node);
+        
+        // Player 1 fold
+        let p1_fold_terminal = NodeData::Terminal(TerminalData::new(15.0));
+        cfr_state.add(player1_idx, 0, p1_fold_terminal);
+        
+        // Player 1 call
+        let p1_call_terminal = NodeData::Terminal(TerminalData::new(5.0));
+        cfr_state.add(player1_idx, 1, p1_call_terminal);
+    }
+    
+    // After raise - player 1 decision
+    let player1_vs_raise = NodeData::Player(PlayerData { regret_matcher: None });
+    let player1_vs_raise_idx = cfr_state.add(player0_raise, 0, player1_vs_raise);
+    
+    // Player 1 fold vs raise
+    let p1_fold_vs_raise = NodeData::Terminal(TerminalData::new(20.0));
+    cfr_state.add(player1_vs_raise_idx, 0, p1_fold_vs_raise);
+    
+    // Player 1 call vs raise - goes to another chance node
+    let chance_after_call_vs_raise = cfr_state.add(player1_vs_raise_idx, 1, NodeData::Chance);
+    
+    // Final terminal node after chance
+    let final_terminal = NodeData::Terminal(TerminalData::new(30.0));
+    cfr_state.add(chance_after_call_vs_raise, 0, final_terminal);
+    
+    // Increment some counts to simulate traversals
+    if let Some(mut node) = cfr_state.get_mut(player0_idx) {
+        node.increment_count(1);  // Call was taken once
+        node.increment_count(2);  // Raise was taken twice
+        node.increment_count(2);
+    }
```

Comment:
Please pull this into a seperate function. `create_example_cfr` or something similar. That will allow users to lean how to export by reading only the main func.